runtime/lib/ttnn/operations/eltwise/unary/unary_composite.h

```diff
@@ -14,6 +14,8 @@ inline bool isUnaryCompositeOp(const ::tt::target::ttnn::EltwiseOp *op) {
   switch (op->type()) {
   case ::tt::target::ttnn::EltwiseOpType::Cbrt:
     return true;
+  case ::tt::target::ttnn::EltwiseOpType::Clamp:
+    return true;
```

Comment:
Nit: Can you merge this switch statement please to return true for all three cases directly?
```
case 1:
case 2:
case 3:
  return true;
```